include/cuco/static_map.cuh

```diff
@@ -761,6 +761,8 @@ class static_map {
                                                   ///< value_type`
     using slot_type = typename device_view_base::slot_type;  ///< Type of hash map slots
 
+    using fetch_insert_result = thrust::pair<iterator, bool>;  ///< Return type of fetch_insert
```

Comment:
I'd avoid an alias here tbh. I think it's clearer to just be explicit about the return type on the function declaration.

Reply:
@sleeepyjack asked for a more descriptive type name in https://github.com/NVIDIA/cuCollections/pull/202#discussion_r938754571. As discussed further, I may have misunderstood the request, so happy to revert back...

Reply:
Please get rid of the alias and directly use `thrust::pair<iterator, bool>`.